### DISCUSSION TITLE
Cap 3 with cordova plugin android error - cordova_not_available

### DISCUSSION THREAD

**Discussion by bertrand85** [ionic-team/capacitor#5310] [Q&A] [answered]
Hi All,

starting a blank capacitor app for testing 
[cordova neutrinos document scanner](https://ionicframework.com/docs/native/document-scanner)

when run on android emulator, it return errors in console

![image](https://user-images.githubusercontent.com/4402449/145673523-378b1fa7-def5-466f-8596-5ec984ccd767.png)

Ionic info 

![image](https://user-images.githubusercontent.com/4402449/145673553-fa07c501-0bf4-4f45-923e-3566528fa78e.png)

Project can be found at [https://github.com/bertrand85/ionic5-cap-cordova](https://github.com/bertrand85/ionic5-cap-cordova)

**Comment by jcesarmobile** [comment, marked as answer]
are you running as web/pwa? (i.e. running from chrome or other web browser)
or as native app? (running from Android Studio)

I've tested your app and I can't reproduce, if I click the button I see the document scanner.

If you are running as web, Cordova plugins are not supported on web
If you are running as native app, make sure you ran `npx cap sync android` and if still get the error try syncing gradle (elephant button)

**Reply by bertrand85** [reply]
Thank', 
I've uninstall all node & android studio.

Reinstall all with JDK 15 and work fine now
